CRM/Report/Form.php

```diff
@@ -361,6 +361,12 @@ class CRM_Report_Form extends CRM_Core_Form {
   public $_orderBy = NULL;
   public $_orderByFields = array();
   public $_orderByArray = array();
+  /**
+   * Array of clauses to group by.
+   *
+   * @var array
+   */
+  protected $_groupByArray = array();
```

Comment:
no reason for it to be public that I can think of - we should also be more restrictive if we can. Also, protected is compatible with extended report